src/main/cpp/systemds.cpp

```diff
@@ -170,17 +158,17 @@ JNIEXPORT jint JNICALL Java_org_apache_sysds_utils_NativeHelper_conv2dDense(
   if(inputPtr == NULL || filterPtr == NULL || retPtr == NULL)
     return (jint) -1;
   
-  int nnz = dconv2dBiasAddDense(inputPtr, 0, filterPtr, retPtr,
-    (int) N, (int) C, (int) H, (int) W, (int) K, (int) R, (int) S,
-    (int) stride_h, (int) stride_w, (int) pad_h, (int) pad_w, (int) P, (int) Q, false, (int) numThreads);
+  size_t nnz = dconv2dBiasAddDense(inputPtr, 0, filterPtr, retPtr, (int) N, (int) C, (int) H, (int) W, (int) K,
+								   (int) R, (int) S, (int) stride_h, (int) stride_w, (int) pad_h, (int) pad_w, (int) P,
+								   (int) Q, false, (int) numThreads);
```

Comment:
Indentation broken

Reply:
Yes, that's a mess. I started to format it for a pleasant visual appearance. Of course that's not going to work out with tabs. So I changed it to two tabs for line continuation (like we have it in java).